src/app/PlaceDetailsData.js

```diff
@@ -131,9 +131,11 @@ function fetchPhotos(
             return [];
           })
       : Promise.resolve([]),
-  ]).then((photoArrays: PhotoModel[][]) => {
-    return [].concat(photoArrays[0], photoArrays[1]);
-  });
+  ])
+    .then((photoArrays: PhotoModel[][]) => {
+      return [].concat(photoArrays[0], photoArrays[1]);
```

Comment:
Eagerly waiting for https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat…